caiman_napari/_mcorr.py

```diff
@@ -36,11 +36,22 @@ def main(batch_path, uuid):
     fnames = [str(input_movie_path)]
     mc = MotionCorrect(fnames, dview = dview, **opts.get_group('motion'))
     mc.motion_correct(save_movie = True)
-    np.save(input_movie_path + 'mc.npy', mc.mmap_file)
-
-def load_output_mcorr(viewer, batch_path, uuid):
+    ix = df[df['uuid'] == uuid].index[0]
+    df['outputs'][ix] = mc.mmap_file
```

Comment:
@ArjunPutcha Does pandas give warnings for this line? I can't remember off the top of my head but I think that setting single dataframe values is not supported with certain types of indexing and `df.loc` needs to be used. Something like:

```
df.loc[df['uuid'] == uuid, 'outputs'] = [mc.mmap_file]
```

Reply:
Also, we want the data in the `ouptuts` column to be a list not a single item.

Reply:
Wait sorry!!! We want a dict of outputs, like I mentioned in a previous comment:

```python
{"mcorr_output": "/path/to/mcorr_vid.mmap",
 "success": True,
 "traceback": None}
```

Reply:
The `success` key will let us set the listwidgetitem background color